docs/developers/ROADMAP_0_3.md

```diff
@@ -0,0 +1,76 @@
+# Roadmap for 0.3 series of releases - April 2020.
+
+The napari roadmap captures current development priorities within the project and should serve as a guide for napari core developers, to encourage and inspire contributors, and to provide insights to external developers who are interested in building for the napari ecosystem. For more details on what this document is and is not, see the [about this document section](#about-this-document).
+
+The [mission](MISSION_AND_VALUES.md#our-mission) of napari is to be a foundational multi-dimensional image viewer for Python and provide graphical user interface (GUI) access to image analysis tools from the Scientific Python ecosystem for scientists across domains and levels of coding experience. To work towards this mission, we have set the following three high-level priorities to guide us over the upcoming months:
+
+- Make the **data viewing and annotation** capabilities **bug-free and fast**.
+
```

Comment:
I didn't see headless mode or serialization items in these goals below.

Reply:
good catches, they should be captured somewhere. curious where @jni thinks they should go?

Reply:
addressed in comment above